app/GraphQL/Ecosystem/Resolvers/AccessControlList/RolesResolver.php

```diff
@@ -0,0 +1,33 @@
+<?php
+declare(strict_types=1);
+namespace App\GraphQL\Ecosystem\Resolvers\AccessControlList;
+
+use Kanvas\AccessControlList\Repositories\RolesRepository;
+use Illuminate\Database\Eloquent\Collection;
+use Kanvas\Users\Repositories\UsersRepository;
+
+class RolesResolver
+{
+    /**
+     * getAllRoles
+     *
+     * @return Collection
+     */
+    public function getAllRoles(): ?Collection
+    {
+        return RolesRepository::getAllRoles();
+    }
+
+    /**
+     * hasRole
+     *
+     * @param  mixed $_
+     * @param  array $request
+     * @return bool
+     */
+    public function hasRole(mixed $_, array $request): bool
+    {
+        $user = UsersRepository::getById($request['userId'], auth()->user()->defaultCompany->id);
+        return $user->isAn($request['role']);
```

Comment:
@FredPeal dont we need to verify the roles exist or the action handles it?

Reply:
If the role not exists the function return false